doc/odml_ontology/root-ontology.ttl

```diff
@@ -0,0 +1,330 @@
+@prefix : <http://webprotege.stanford.edu/project/9IqNfOthyW8sqU6S3HagXc#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@base <http://webprotege.stanford.edu/project/9IqNfOthyW8sqU6S3HagXc> .
+
+<http://webprotege.stanford.edu/project/9IqNfOthyW8sqU6S3HagXc> rdf:type owl:Ontology .
+
+#################################################################
+#    Datatypes
+#################################################################
+
+###  http://www.w3.org/1999/02/22-rdf-syntax-ns#Bag
+rdf:Bag rdf:type rdfs:Datatype ;
+        rdfs:label "rdf:Bag" .
+
+
+###  http://www.w3.org/1999/02/22-rdf-syntax-ns#li
+rdf:li rdf:type rdfs:Datatype ;
+       rdfs:label "rdf:li" .
+
+
+###  http://www.w3.org/1999/02/22-rdf-syntax-ns#resource
+rdf:resource rdf:type rdfs:Datatype ;
+             rdfs:label "rdf:resource" .
+
+
+###  http://www.w3.org/2001/XMLSchema#dat
+xsd:dat rdf:type rdfs:Datatype ;
```

Comment:
Do we have any idea where this comes from? I don't see it come up anywhere in Webprotege and I also don't see it actually being used anywhere.